rllib/utils/runners/runner.py

```diff
@@ -81,6 +85,17 @@ def stop(self) -> None:
         that its `close()` method is called.
         """
 
+    @property
+    @abc.abstractmethod
+    def _device(self) -> DeviceType:
+        """Returns the device of this `Runner`."""
+        pass
+
+    @abc.abstractmethod
+    def set_device(self) -> None:
+        """Sets the device for this `Runner`."""
```

Comment:
Let's add more detail here as to who this method would be used/overridden?
Dumb question: Does the subclass infer the correct device through `self.config`?